build.include

```diff
@@ -104,11 +109,20 @@ publishImages() {
     docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${IMAGE_TAG}"
     echo y | docker push "${REGISTRY}/${image}:${IMAGE_TAG}"
     if [[ -n "${THEIA_DOCKER_IMAGE_VERSION}" ]]; then
-      echo "Publishing ${image}:${THEIA_DOCKER_IMAGE_VERSION}..."
-      docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
-      echo y | docker push "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
+      if [[ -n "${ARCH}" ]]; then
+         echo "Publishing ${image}:${THEIA_DOCKER_IMAGE_VERSION}-${TRAVIS_TAG}-${ARCH}..."
+         docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}-${TRAVIS_TAG}-${ARCH}"
+         echo y | docker push "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}-${TRAVIS_TAG}-${ARCH}"
+      else
+         echo "Publishing ${image}:${THEIA_DOCKER_IMAGE_VERSION}..."
+         docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
+         echo y | docker push "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
+      fi
```

Comment:
I don't see why we would require a special case if $ARCH is defined there

Reply:
see above -- it could be blank by default, and still blank if not set with --arch flag.

Reply:
I mean, we don't need to duplicate 3 entire lines if we just change the suffix
We do the same steps so it's just the final image tag that is changed

Reply:
In order to not conflict with the current build flow, the ARCH variable is introduced. For native builds, while executing build.sh we pass --arch flag in case of Travis. 


If we don't specify the image tag as ${THEIA_DOCKER_IMAGE_VERSION}-${TRAVIS_TAG}-${ARCH}, the images pushed to quay.io will be overridden on each arch build.

Reply:
but why do we need ARCH.

Arch is the system where we build the image and it's the same Dockerfile.

so if it's just to carry out a different suffix, why we don't say we build image with tag `next-amd64` from travis script so the image built by docker build on amd64 is named `next-amd64`